contrib/systemd-with-workers/README.md

```diff
@@ -0,0 +1,36 @@
+# Setup Synapse with Workers and Systemd
+
+This is a setup for using managing synapse with systemd including support for managing workers. It provides a `matrix-synapse-homeserver`, as well as a `matrix-synapse-worker@` service for any workers you require. Additionally to group the required services it sets up a `matrix.target`. You can of course add any bot- or bridge-services to this target as well by setting the respective `WantedBy` value in the `[Install]` section of their unit files to `matrix.target`.
```

Comment:
please can you wrap your paragraphs at 80 characters?

Reply:
Of course. Will do this afternoon/evening.